tests/UnifiedSpecTests/EventObserver.php

```diff
@@ -227,7 +226,7 @@ private function assertEvent($actual, stdClass $expected, string $message)
     {
         assertIsObject($actual);
 
-        switch (get_class($actual)) {
+        switch ($actual::class) {
```

Comment:
Use `match` operator here.

Reply:
Updated. I also removed the `return` statements as the methods being called don't return a value. Still, the `match` operator has a much more concise syntax than `switch`, so I think this is an improvement.